Library/DiscUtils.Ntfs/Bitmap.cs

```diff
@@ -201,6 +201,16 @@ public byte GetByte(long index)
             return 0;
         }
 
+        public int GetBytes(byte[] buffer, long index, int count)
```

Comment:
We CANNOT have a public method formed like this. I would assume you mean the Index of the buffer, and not the index of the underlying structure.